### PR TITLE
fix(chat): clean error message when select folder closed (Issue #1337)

### PR DESCRIPTION
**Pull request by likeamike**
**Description:**

Clear the error message when a selected folder is about to close

Issues:

- Issue #1337 

**UI changes**

<Please, provide Screenshots or Figma links>

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1392.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Files/SelectFolderModal.tsx [hunk 1]
```diff
@@ -129,7 +137,7 @@ export const SelectFolderModal = ({
       </SelectFolderHeader>
       <SelectFolderFooter
         handleNewFolder={handleNewFolder}
-        onSelectFolderClick={() => onClose(selectedFolderId)}
+        onSelectFolderClick={() => handleClose(selectedFolderId)}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Files/SelectFolderModal.tsx:140]: Are you sure that we need to clear error message here?

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1392.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Files/SelectFolderModal.tsx (hunk 1 above)
Comment (likeamike) [reply apps/chat/src/components/Files/SelectFolderModal.tsx:140]: Yes, I believe so. Selecting a folder also closes the window and should clear the error message, just as simply closing the window does. In both of these scenarios, we don't want the same error message to appear when we open the window the next time.

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1392.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1392.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1392.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**